lom/Makefile

```diff
@@ -16,6 +16,7 @@ BUILD_DIR := build
 LIB_DIR := $(BUILD_DIR)/lib
 BIN_DIR := $(BUILD_DIR)/bin
 TEST_DIR := $(BUILD_DIR)/test
+CONFIG_DIR :=  $(BUILD_DIR)/config
```

Comment:
You got to update master Makefile, which is looking in config subdir.
Hmm. May be we need to include LoM makefile, so we don't need duplicate declaration. See if we can do this and have one declaration in main Make.